core_data_modules/traced_data/util/fold_traced_data.py

```diff
@@ -185,7 +185,7 @@ def list_of_labels(code_scheme, x, y):
 
         # Ensure that all the codes in each list belong to the code_scheme
         for label in x + y:
-            assert label["SchemeID"] == code_scheme.scheme_id
+            assert label["SchemeID"] == code_scheme.scheme_id, f"{label['SchemeID']} == {code_scheme.scheme_id}"
```

Comment:
Thanks, I think this could be even clearer with e.g.
f"Label scheme id {label...} != code scheme id {code_scheme...}"

Reply:
Thanks was conflicted on != or ==. Updated